parsel/selector.py

```diff
@@ -139,9 +139,9 @@ class Selector(object):
     selectorlist_cls = SelectorList
 
     def __init__(self, text=None, type=None, namespaces=None, root=None,
-                 base_url=None, _expr=None):
+                 base_url=None, _expr=None, parser_cls=None):
```

Comment:
Are there other use cases for this argument other than passing etree.HtmlParser?